src/getHumanAge.test.js

```diff
@@ -4,7 +4,26 @@ describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test('should return an array', () => {
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
```

Comment:
generally ok, but here we also test the logic, not only if it returns an array
but it's not critical - you may leave it as it is